gcloud-java-contrib/gcloud-java-nio/src/main/java/com/google/gcloud/storage/contrib/nio/CloudStorageConfiguration.java

```diff
@@ -103,6 +103,7 @@ public Builder blockSize(int value) {
       return this;
     }
 
+
     /** Creates a new instance, but does not destroy the builder. */
     public CloudStorageConfiguration build() {
```

Comment:
We should avoid oneline javadoc comments (here and in other places).
Also, this class java doc descripiton is missing a period.